README.md

```diff
@@ -1,2 +1,162 @@
-# asgardeo-nextjs-b2b-sample-app
```

Comment:
It'd be better to keep a title and a one line description at the top of the README, IMO

Reply:
Fixed